app/views/users/index.html.erb

```diff
@@ -1 +1,21 @@
-<h1>Here is a list of all users</h1>
+<br/>
+<div class="container-sm">
+  <% @users.each do |user|%>
+    <div class="card mb-3">
+      <div class="row g-0">
+        <div class="col-md-4" style="width: 100px;">
+          <a href = "<%= user_posts_path(user) %>">
+            <img src =<%= user.photo %> class="img-fluid rounded" alt="User Photo">
+          </a>
+        </div>
+        <div class="col-md-8">
+          <div class="card-body">
+            <h5 class="card-title"><%= link_to user.name, user_path(user) %></h5>
+            <p class="card-text"><small class="text-muted">Number of posts: <%= user.post_counter %></small></p>
+          </div>
+        </div>
```

Comment:
- [ ] Amazing job so far but I think it would be great if you place the number of posts to the right as shown in the given wireframe so please add it to be the same ⬇✅💯💯

- [ ] I think it would be great if you make the counter works to count the number of posts so I could see the number of posts instead of zer since here I inserted one post so I should see `1` instead of `0` so kindly you can use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).  as mentioned in the requirements 💯🔥

- [ ] Kindly make a border to the image and place it beside the name box so it should be the same given wireframe 💪🏻

Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

 - Yours
![image](https://github.com/noelfoka/Blog_app/assets/58553711/3f1597fe-1e42-4972-9f33-e84d05267021)


- Given
![image](https://github.com/noelfoka/Blog_app/assets/58553711/e5e27858-1493-4250-b051-4439d7bdcb5e)

Reply:
This change has been done well